### PR TITLE
correct some English writing errors

### PR DESCRIPTION
**Pull request by jtao1735**

### CODE REVIEW

File: documentation20/en/00.index/docs.md [hunk 1]
```diff
@@ -1,37 +1,50 @@
...
+* [Deploy to Kubernetes](https://taosdata.github.io/TDengine-Operator/en/index.html)：a detailed guide for TDengine deployment in Kubernetes environment
 
 ## [Overall Architecture](/architecture)
 
-- [Data Model](/architecture#model): relational database model, but one table for one device with static tags
-- [Cluster and Primary Logical Unit](/architecture#cluster): Take advantage of NoSQL, support scale-out and high-reliability
-- [Storage Model and Data Partitioning/Sharding](/architecture#sharding): tag data will be separated from time-series data, segmented by vnode and time
-- [Data Writing and Replication Process](/architecture#replication): records received are written to WAL, cached, with acknowledgement is sent back to client, while supporting multi-replicas
+- [Data Model](/architecture#model): relational database model, but one table for one data collection point with static tags
+- [Cluster and Primary Logical Unit](/architecture#cluster): Take advantage of NoSQ architecture, highly available and horizontal scalable
```
Comment (sangshuduo) [inline documentation20/en/00.index/docs.md]: NoSQL

Comment (sangshuduo) [inline documentation20/en/00.index/docs.md]: how about high-availability (noun)?

Comment (sangshuduo) [inline documentation20/en/00.index/docs.md]: how about scalability (noun)?

File: documentation20/en/02.getting-started/docs.md [hunk 2]
```diff
@@ -2,21 +2,19 @@
...
+TDengine software consists of 3 parts: server, client, and alarm module. At the moment, TDengine server only runs on Linux (Windows, mac OS and more OS supports will come soon), but client can run on either Windows or Linux. TDengine client can be installed and run on Windows or Linux. Applications based-on any OSes can all connect to server taosd via a RESTful interface. About CPU, TDengine supports X64/ARM64/MIPS64/Alpha64, and ARM32、RISC-V, other more CPU architectures will be supported soon. You can set up and install TDengine server either from the [source code](https://www.taosdata.com/en/getting-started/#Install-from-Source) or the [packages](https://www.taosdata.com/en/getting-started/#Install-from-Package).
```
Comment (sangshuduo) [inline documentation20/en/02.getting-started/docs.md:5]: alert module?

Comment (sangshuduo) [inline documentation20/en/02.getting-started/docs.md:5]: ... ARM32, RISC-V,  ...

**Review by sangshuduo — COMMENTED** [review]
I found few typo. Please see my comments.

File: documentation20/en/00.index/docs.md [hunk 3]
```diff
@@ -1,37 +1,50 @@
...
+TDengine is a highly efficient platform to store, query, and analyze time-series data. It is specially designed and optimized for IoT, Internet of Vehicles, Industrial IoT, IT Infrastructure and Application Monitoring, etc. It works like a relational database, such as MySQL, but you are strongly encouraged to read through the following documentation before you experience it, especially the Data Modeling sections. In addition to this document, you should also download and read the technology white paper. For the older TDengine version 1.6 documentation, please click [here](https://www.taosdata.com/en/documentation16/).
 
 ## [TDengine Introduction](/evaluation)
 
 * [TDengine Introduction and Features](/evaluation#intro)
 * [TDengine Use Scenes](/evaluation#scenes)
-* [TDengine Performance Metrics and Verification](/evaluation#)
+* [TDengine Performance Metrics and Verification]((/evaluation#))
```
Comment (Aries-Lee1991) [inline documentation20/en/00.index/docs.md:9]: two pairs of parentheses?

Comment (Aries-Lee1991) [inline documentation20/en/00.index/docs.md]: two pairs of parentheses?

**Review by Aries-Lee1991 — COMMENTED** [review]
see my comments

**Pull request merged**